lib/stream/leo-stream.js

```diff
@@ -230,15 +230,16 @@ module.exports = function(configure) {
 				done(null, e);
 			}));
 			if (opts.useS3 && !opts.firehose) {
-				args.push(leoS3(ls, outQueue, configure, {prefix: id}));
-			} else {
+				args.push(leoS3(ls, outQueue, configure, {prefix: opts.prefix || id}));
+			} else if (!opts.firehose) {
```

Comment:
when using firehose it is bad to send s3 command events created by this code.  1) firehose is going to s3 anyway 2) the resulting command event is compressed and the firehose processor doesn't know how to handle it